app/server/utils.py

```diff
@@ -433,6 +433,24 @@ def paint(self, documents):
             data.append(d)
         return data
 
+    def paint_labels(self, documents, labels):
+        serializer_labels = LabelSerializer(labels, many = True)
```

Comment:
Formatting nit: no space between named argument and value (`many=True`, not `many = True`).